packages/plugin-core/assets/dendron-ws/tutorial/tutorial.3-links-and-graphs.md

```diff
@@ -0,0 +1,45 @@
+---
+id: hAB2tQHsi0CGuhxz5HkTI
+title: Links and Graphs
+desc: ''
+updated: 1624455055929
+created: 1624333278136
+---
+
+### Links
+
+Dendron supports links between your notes, which can help you organically link your thoughts and build up a powerful knowledge graph.
```

Comment:
which is an all organic way to link your thoughts and build up a powerful knowledge graph